### DISCUSSION TITLE
Managing Scene Objects Dynamically

### DISCUSSION THREAD

**Discussion by Kapi2910** [NVlabs/sionna#632] [Q&A] [answered]
Is there a way to add mitsuba objects dynamically.  I have a base scene and in it I want to instantiate a random number of objects, possibly moving. For each instantiation, I want to find the coverage map.

**Comment by merlinND** [comment, marked as answer]
Hello @Kapi2910,

Please have a look at this pull request: https://github.com/NVlabs/sionna/pull/536
